src/TreeViewList/TreeViewList.tsx

```diff
@@ -267,13 +267,20 @@ const TreeViewList = ({
               })}
             ></Box>
             {enableSearch ? (
-              <SearchBar
-                value={searchValue}
-                onChange={event => {
-                  setSelectedNode("");
-                  setSearchValue(event.target.value);
-                }}
-              />
+              <Box
+                sx={theme => ({
+                  marginBottom: theme.spacing(2),
+                  marginTop: theme.spacing(2)
+                })}
```

Comment:
Please be consistent with your styling. I suggest you get rid of theme here
```suggestion
                sx={my: 2}
```